source/jormungandr/jormungandr/interfaces/v1/Places.py

```diff
@@ -247,7 +247,8 @@ def get(self, region=None, lon=None, lat=None):
             response = i_manager.dispatch(args, "places", instance_name=instance)
         else:
             if global_autocomplete:
-                bragi_response = global_autocomplete.get(args, None)
+                user = authentication.get_user(token=authentication.get_token())
```

Comment:
get_user may raise or return None. So try around that, but you'll have to check that it's not None also.